src/ButtonThread.h

```diff
@@ -205,6 +205,16 @@ class ButtonThread : public concurrency::OSThread
         }
     }
 
+    static meshtastic_Config_PositionConfig_GpsMode toggleGpsMode(meshtastic_Config_PositionConfig_GpsMode currentMode)
```

Comment:
I'd rather not put this GPS related method into this ButtonThread class. Doesn't it belong to NodeDB for determining the default/migrated state?